cygnus-ngsi/src/test/java/com/telefonica/iot/cygnus/sinks/NGSICartoDBSinkTest.java

```diff
@@ -179,6 +194,64 @@ public void testConfigureFlipCoordinatesOK() {
     } // testConfigureFlipCoordinatesOK
     
     /**
+     * [NGSICartoDBSink.configure] -------- Configured `enable_raw` cannot be different than `true` or `false`.
+     */
+    @Test
+    public void testConfigureEnableRawOK() {
+        System.out.println(getTestTraceHead("[NGSICartoDBSink.configure]")
+                + "-------- Configured `enable_raw` cannot be different than `true` or `false`");
+        String endpoint = "https://localhost";
+        String apiKey = "1234567890abcdef";
+        String dataModel = null; // default one
+        String enableLowercase = null; // default one
+        String flipCoordinates = null; // default value
+        String enableRaw = "falso";
+        String enableDistance = null; // default one
+        NGSICartoDBSink sink = new NGSICartoDBSink();
+        sink.configure(createContext(endpoint, apiKey, dataModel, enableLowercase, flipCoordinates, enableRaw,
+                enableDistance));
+        
+        try {
+            assertTrue(sink.invalidConfiguration);
+            System.out.println(getTestTraceHead("[NGSICartoDBSink.configure]")
+                    + "-  OK  - 'enable_raw=falso' was detected");
+        } catch (AssertionError e) {
+            System.out.println(getTestTraceHead("[NGSICartoDBSink.configure]")
+                    + "- FAIL - 'enable_raw=falso' was not detected");
+            throw e;
+        } // try catch
+    } // testConfigureEnableRawOK
+    
+    /**
+     * [NGSICartoDBSink.configure] -------- Configured `enable_distance` cannot be different than `true` or `false`.
+     */
+    @Test
+    public void testConfigureEnableDitanceOK() {
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1031/commits/7e16f2ded5694f3629e1416296981a6dcd4ec061